proto/metadata.proto

```diff
@@ -929,6 +929,9 @@ message Metadata {
     // components for Nokia and Arista.
     bool line_port_unsupported = 322;
 
+     // Devices that do not support configuring a two rate three color policer through OC
```

Comment:
There must be a url linking to an issue tracking this deviation in the format: `https://partnerissuetracker.corp.google.com/issues/xxxxxx`